students/12th/heungsoolee/js/login.js

```diff
@@ -0,0 +1,41 @@
+const inputId = document.getElementsByClassName("id_input")[0];
+const inputPassword = document.getElementsByClassName("password_input")[0];
+const loginBtn = document.getElementsByClassName("login_button")[0];
+
+const slideImg = document.getElementsByClassName("slide_img");
+const slideClassName = "slide_now";
+
+const changeLoginBtn = () => {
+  let keyValue = event.key;
+
+  if (keyValue.length > 0) {
+    if (inputPassword.value.length > 0 && inputId.value.length > 0) {
+      loginBtn.style.backgroundColor = "#0095F6";
+      loginBtn.style.cursor = "pointer";
+    } else {
+      loginBtn.style.backgroundColor = "#C0DFFD";
+      loginBtn.style.cursor = "";
+    }
+  }
+};
+
+const slideShow = () => {
+  let nowImg = document.getElementsByClassName(slideClassName)[0];
```

Comment:
```suggestion
  const nowImg = document.getElementsByClassName("slide_now")[0];
```
- 특별히 변수로 지정할 부분이 아닌 것 같습니다
- 특별한 이유 없으면 변수는 const로 선언해주세요